models/calendar.yml

```diff
@@ -319,10 +319,17 @@
         title: Introductions
         emoji: 👋
 
-    - 7:45 PM:
+    - 7:45 PM:  
         type: Talk
         emoji: 🎨
-        title: TBD
+        title: 'Tech Feminism 101: Why We Should All Be Feminists'
+        description: '2016 has been a good year for Feminism and our 
```

Comment:
Sorry, but would you mind formatting it in this way?

```yml
description: |
  2016 has been a good year for Feminism and our...
```

Right now the `'` in `But if it's been a` is breaking the quote-wrapping trick 😅  But the `|` should make it work!